### ISSUE TITLE
[jvm-packages] the result dataframe with multi:softmax obj function should not contain probability col

### ISSUE THREAD

**Issue by CodingCat** [dmlc/xgboost#3506]
according to https://github.com/dmlc/xgboost/blob/a725272e19748a5f3ddd7eeeff7c766660e38a06/src/objective/multiclass_obj.cc#L94-L120

we should not have probability col in multi:softmax case and instead only rawPrediction column and prediction 

otherwise, probability would be filled with prediction and no real probability would be presented
